### PR TITLE
zombienet runs devnet all and in devnet container well #cu-862j1xn5b

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
- removed karura only devnet (not run directly ever, only as part of ALL)
- DALI container and ALL ZN run well
- removed not used files (these are generated with ALL script now)
- prepared some container tools for reuse in bridge arion (moved them into tools)

### CODE REVIEW

File: inputs/paritytech/zombienet/flake-module.nix [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        rev = "b9f089eb55a7cb1d4c12575b8323cb9b9fab4a60";
+        hash = "sha256-EC6XKbcI+Is0RGlmC8WGjPqiFh9Ulf3bXDoVihtYqsU=";
+      };
+      paritytech-zombienet = pkgs.stdenv.mkDerivation {
+        name = "zombienet";
+        src = "${paritytech-zombienet-src}/javascript";
+        buildInputs = with pkgs; [ nodejs ];
+        nativeBuildInputs = with pkgs;
+          [
+            yarn
```
Comment (obsessed-cake) [inline inputs/paritytech/zombienet/flake-module.nix:16]: Is it exactly needed? You use `npm` to build `zombienet`.

**Review by obsessed-cake — APPROVED** [review]

File: inputs/paritytech/zombienet/flake-module.nix (hunk 1 above)
Comment (dzmitry-lahoda) [reply inputs/paritytech/zombienet/flake-module.nix:16]: they have adhoc custom build with `so` deps (rust stuff), that just does not builds/runs in nix that simple. so need to try again later

Comment (dzmitry-lahoda) [reply inputs/paritytech/zombienet/flake-module.nix:16]: so answers to be clear-  i tried to run build, it failed with some nasty error. so need to try again as some point.

**Review by KaiserKarel — APPROVED** [review]
It's a bit above my nix level, but looks good

**Review by cor — APPROVED** [review]
LGTM! Nice work and I like how you made `zombienet` a directory. Seems like a good solution for when an input has "supporting files"

**Comment by dzmitry-lahoda** [comment]
cc @andyjsbell

File: inputs/paritytech/zombienet/flake-module.nix (hunk 1 above)
Comment (dzmitry-lahoda) [reply inputs/paritytech/zombienet/flake-module.nix:16]: https://app.clickup.com/t/20465559/NX-67

**Pull request merged**